src/webchat-ui/components/presentational/Textarea.tsx

```diff
@@ -0,0 +1,30 @@
+import tinycolor from 'tinycolor2';
+import { styled } from "../../style";
+import { interactionCss, createTransition } from '../../utils/css';
+
+export default styled.textarea(({ theme }) => ({
+	...interactionCss,
+	resize: "none",
+	boxSizing: 'border-box',
+	borderRadius: theme.cornerSize,
+	borderColor: 'transparent',
+	borderWidth: 2,
+	borderStyle: 'solid',
+	outline: 'none',
+	padding: `${theme.unitSize}px ${theme.unitSize * 2}px`,
+	backgroundColor: tinycolor('white').setAlpha(.7).toHslString(),
```

Comment:
Good thing we have tinycolor for that :D Couldn't imagine how to write that as a css color string...

Reply:
I copied over the values from Input.tsx to keep the styling in sync. Would this color not be just rgba(255,255,255, 0.7) ?

Reply:
Not that important :)